client/src/RatingsAndReviews/ReviewsClicker.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
higher order component looks super clean!  i noticed that you were using e.target.attribute on yours... i was doing same thing but checking each classname, which was unnecessary - so thanks for helping me clear like 50 lines in click component 🔥